.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Launch",
+			"type": "go",
+			"request": "launch",
+			"mode": "debug",
+			"remotePath": "",
+			"port": 2345,
+			"host": "127.0.0.1",
+			"program": "${workspaceRoot}/cmd/platform",
+			"env": {},
+			"args": [],
+			"showLog": true
+		}
+	]
+}
```

Comment:
Ya. It's for vscode. Corey uses it as well so i though it would be useful to check in.